numfun/polyfit.py

```diff
@@ -15,28 +18,28 @@ def polyfit_jit(x: np.array, y: np.array, n: int, a: float, b: float) -> tuple:
     :return: the output of np.linalg.lstsq()
     """
     m = len(x)
-    Tx = np.zeros((m, n + 1))
-    Tx[:, 0] = np.ones(m)
+    tx = np.zeros((m, n + 1))
+    tx[:, 0] = np.ones(m)
     x_map = 2.0 * (x - a) / (b - a) - 1.0
```

Comment:
Can you please continue to use Tx instead of tx? The convention used in numerical linear algebra is that any matrix or 2-D array is in capital letters while the 1-D arrays and vectors are in small letters